library/std/src/process.rs

```diff
@@ -227,9 +215,8 @@ impl fmt::Debug for Child {
 /// file handle will be closed. If the child process was blocked on input prior
 /// to being dropped, it will become unblocked after dropping.
 ///
-/// [`Child`]: struct.Child.html
-/// [`stdin`]: struct.Child.html#structfield.stdin
-/// [dropped]: ../ops/trait.Drop.html
+/// [`stdin`]: crate::process::Child.stdin
```

Comment:
This part isn't working. I tried `Child.stdin` too, but that didn't work either. What should I do to get it to work?

Reply:
Try `Child::stdin` instead. Looks like this isn't documented on https://doc.rust-lang.org/nightly/rustdoc/unstable-features.html#linking-to-items-by-name, @Manishearth are you interested in fixing that?

As a side note the syntax for this is a little confusing ... maybe we should support both `::` and `.` for fields?

Reply:
Hmm, that's weird – [the RFC says](https://github.com/rust-lang/rfcs/blob/68e17ace829e81239fe5020b7a6dbc22550b6d16/text/1946-intra-rustdoc-links.md#linking-to-fields) to use `Struct.field` to reference a struct field:

> ## Linking to Fields
>
> To link to the fields of a struct we propose to write the path to the struct, followed by a dot, followed by the field name.
>
> For example:
>
> ```markdown
> This is stored in the [`size`](storage::Filesystem.size) field.
> ```

Reply:
Yeah the implementation is out of sync with the RFC in some places, I guess this is one of them. I'll open an issue.

Reply:
Opened https://github.com/rust-lang/rust/issues/75437.